package.json

```diff
@@ -147,7 +153,9 @@
     "linthtml": "htmlhint media src",
     "unittest": "node ./out/Tests/runTest.js",
     "test": "npm run unittest",
-    "clean": "rm -rf out"
+    "clean": "rm -rf out",
+    "copyfile": "copyfiles --error --up 1 src/**/*.py out/",
+    "postcompile": "npm run copyfile"
```

Comment:
These scripts need to be considered more. That could be
```json
{
  "postcompile": "node ./out/runCopyFiles.js" // runCopyFiles describes what files are copied.
}
```

But, what else way? like ... without copying python files?

Reply:
I haven't considered about this approach. It could make the package more lighter. (right? sorry I'm so newbie in ts/js project)